test/benchmarks/mongoBench/runner.js

```diff
@@ -190,8 +190,8 @@ class Runner {
 
   _errorHandler(error) {
     this.reporter(`Error: ${error.name} - ${error.message} - ${error.stack}`);
-    for (let error = error.cause; error != null; error = error.cause) {
-      this.reporter(`Caused by: ${error.name} - ${error.message} - ${error.stack}`);
+    for (let cause = error.cause; cause != null; cause = cause.cause) {
```

Comment:
Fixed a bug here, `let error` conflicted with the existing error param throwing a ReferenceError